### ISSUE TITLE
Blank lines are not drawn in fresh terminal if they are beyond line 24 (hotfixed in v0.1.0)

### ISSUE THREAD

**Issue by Wramberg** [Wramberg/TerminalView#12]
This is due to the internal pyte terminal being initialized with size 80, 24. This triggers an initialization of lines 1 to 24. When it is subsequently resized (since very few have a terminal view of size 80, 24) it does not trigger a redraw of the newly added lines which renders them empty (as in completely empty not blank like we want).

When we subsequently receive data pyte assumes the lines are blank though. So if the shell outputs:
```
27: This is line followed by blank
28: 
29: Last line after blank
```
Pyte simply triggers us to draw line 27 and 29 thinking line 28 is blank - but at this point we dont know anything about line

**Comment by Wramberg** [comment]
A quickfix is to just initialize the internal pyte terminal a lot larger. Ill leave this open though as better handling would probably be preferred.

I'm thinking something where we trigger a redraw or reset of the terminal on resize.

**Comment by Wramberg** [comment]
Fixed in baeef2b92840a27258e4c664c96a892af619befb
Will be included in next release.
